ctru-rs/src/services/gfx.rs

```diff
@@ -14,6 +14,7 @@ use crate::services::ServiceReference;
 
 mod private {
     use super::{BottomScreen, TopScreen, TopScreen3D, TopScreenLeft, TopScreenRight};
+    use crate::console::Console;
```

Comment:
If we are planning on putting more structs into this field, we might have to consider moving this into `lib.rs` or into a separate module.

Reply:
Moved to separate file